cinema/views.py

```diff
@@ -101,8 +104,26 @@ def get_serializer_class(self):
         if self.action == "retrieve":
             return MovieDetailSerializer
 
+        if self.action == "upload_image":
+            return MovieImageSerializer
+
         return MovieSerializer
 
+    @action(
+        detail=True,
+        methods=["POST"],
+        permission_classes=(IsAdminUser,),
+        url_path="upload_image",
```

Comment:
Make a following change
```suggestion
        url_path="upload-image",
```